### PR TITLE
Add missing dependency to python-dev and dynamically find host in jav…

### PR DESCRIPTION
**Pull request by mortalisk**
…ascript

### CODE REVIEW

File: friskby/static/js/friskby.js [hunk 1]
```diff
@@ -89,13 +89,15 @@ var FriskbyModuleExtensions = (function ( Friskby ) {
...
+        var host = location.host;
+
         var apiUrl = {
-            sensor:             "http://friskby.herokuapp.com/sensor/api/sensor/",
-            sensorType:         "http://friskby.herokuapp.com/sensor/api/sensortype/",
-            sensorInfo:         "http://friskby.herokuapp.com/sensor/api/sensorinfo/",
-            device:             "http://friskby.herokuapp.com/sensor/api/device/",
-            location:           "http://friskby.herokuapp.com/sensor/api/location/",
-            measurementType:    "http://friskby.herokuapp.com/sensor/api/measurement_type/"
+            sensor:             "http://" + host + "/sensor/api/sensor/",
```
Comment (joakim-hove) [inline friskby/static/js/friskby.js]: Hmmm - in my head I was thinking we could replace this with a relative url:
```html
sensor : "/sensor/api/sensor/",
sensorType: "/sensor/api/snesortype/"
```

but that might have been just whishful thinking?

File: README.md [hunk 2]
```diff
@@ -22,10 +22,10 @@ or even Linux specific to the software as such.
...
 You need to install the following packages: git, postgresql,
-postgresql-server-dev-all and python-pip:
+postgresql-server-dev-all, python-dev and python-pip:
 
 ```bash
-bash% sudo apt-get install git postgresql postgresql-server-dev-all python-pip
+bash% sudo apt-get install git postgresql postgresql-server-dev-all python-pip python-dev
 ```
 
 After pip has been installed, we install all the required Python packages by
```
Comment (joakim-hove) [inline README.md]: I agree with @pgdr - the `README.md` and `settings/__init__.py` changes should be in a separate PR; and I will merge them right away.

File: friskby/static/js/friskby.js (hunk 1 above)
Comment (pgdr) [inline friskby/static/js/friskby.js]: I think we should do absolute URLs, but with time, they will be generated by django, and thus we will not need to care about them in js at all.

**Comment by pgdr** [comment]
If Joakim does not object, I suggest we merge this now.

**Comment by joakim-hove** [comment]
> I can also accept to merge if DEBUG..., but maybe we can have an own DevSettings which has allowed hosts equal *?

I agree that allowing all hosts should be limited to a debug situation. 

> but maybe we can have an own DevSettings which has allowed hosts equal *?

My very clear experience with django settings is that it is best to have *only one* settings, and then manipulate that through the use of environment variables. For the particular case we would get the all allowed hosts behavior with:

```python
if DEBUG:
   ALLOWED_HOSTS = ["*"]
else:
   ALLOWED_HOSTS=["friskby.herokuapp.com"]
```

and then setting the environment variable `FRISKBY_DEBUG` before invoking the local server.

```bash
export FRISKBY_DEBUG=True
```

**Comment by pgdr** [comment]
Good work!  Thanks!

**Pull request merged**